docs/pyproject.toml

```diff
@@ -1,20 +1,19 @@
 [tool.poetry]
 name = "scylla"
-description = "Scylla Documentation"
-version = "4.3.0"
+description = "ScyllaDB Documentation"
+version = "0.1.0"
```

Comment:
Why does version change from 4.3 to 0.1?

Reply:
When installing the docs locally, it shows "Installing the current project: ... (4.3.0)" because the ScyllaDB version was 4.3.0 when the toolchain was installed. We updated the version in `pyproject.toml` to avoid confusion, as we won't be releasing the docs as a Python package.